test/torchaudio_unittest/prototype/hdemucs_test_impl.py

```diff
@@ -0,0 +1,76 @@
+import torch
+from parameterized import parameterized
+from torchaudio.prototype.models.hdemucs import HDemucs, _HEncLayer, _HDecLayer
+from torchaudio_unittest.common_utils import TestBaseMixin, torch_script
+
+_SOURCE_SETS = [
+    (["bass", "drums", "other", "vocals"],),
+    (["bass", "drums", "other"],),
+    (["bass", "vocals"],),
+    (["vocals"],),
+]
+
+
+def _get_hdemucs_model(sources):
+    return HDemucs(sources)
+
+
+class TorchscriptConsistencyMixin(TestBaseMixin):
+    r"""Mixin to provide easy access assert torchscript consistency"""
+
+    def _assert_torchscript_consistency(self, model, tensors):
+        ts_func = torch_script(model)
+
+        torch.random.manual_seed(40)
+        output = model(*tensors)
+
+        torch.random.manual_seed(40)
+        ts_output = ts_func(*tensors)
+
+        self.assertEqual(ts_output, output)
+
+
+class HDemucsTests(TorchscriptConsistencyMixin):
```

Comment:
You should be able to just use `TestBaseMixin` here and remove `TorchscriptConsistencyMixin` above since we're not testing torchscript and using the function implemented above